e2e/pierced-react/vite.config.ts

```diff
@@ -13,6 +13,10 @@ if (process.env.NODE_ENV === "development") {
 // https://vitejs.dev/config/
 export default defineConfig({
 	plugins: [react(), wranglerPagesDevWithReload()],
+	build: {
+		minify: false,
+		assetsDir: "pierced-react/assets",
```

Comment:
`assetsDir` change necessary to avoid conflict between pierced-react and dashboard over where /assets/ scripts get loaded from..

`minify` disabled to make stepping through reframed logic easier